src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OpenTelemetry.Exporter.OpenTelemetryProtocol.csproj

```diff
@@ -19,6 +19,7 @@
     <PackageReference Include="Grpc" Version="$(GrpcPkgVer)" Condition="'$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'net462'" />
     <PackageReference Include="Google.Protobuf" Version="$(GoogleProtobufPkgVer)" />
     <PackageReference Include="Grpc.Tools" Version="$(GrpcToolsPkgVer)" PrivateAssets="all" />
+    <PackageReference Include="OpenTelemetry.Extensions.PersistentStorage.Abstractions" Version="$(PersistentStoragePkgVer)" />
```

Comment:
we need to find alternate ways to achieve this. This dependency is unlikely to be accepted for this component, as we have strict 1.5 timeline (when this component need to ship stable) and cannot be blocked as this dependency is a beta component.

Reply:
We should be able to ship stable version for the abstractions before that. Just need to rename the package
https://github.com/open-telemetry/opentelemetry-dotnet/issues/3469

Reply:
Should the main/core/SDK package really on the contrib package?

Reply:
This was discussed in the SIG meeting yesterday, and the general consensus is not to have this dependency.
It could be a separate OTLPWithPersistence package that does this and internally using/overriding the main OTLP Exporter.
Or
OTLP Exporter could expose some general purpose callbacks which can be leveraged by the OTLPWithPersistence package.